external/fv3fit/fv3fit/_shared/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .config import ModelTrainingConfig, load_training_config
+from .config import ModelTrainingConfig, load_training_config, register_keras_trainer
```

Comment:
Should it be called "keras" if it's in shared? It seems broader in scope.  I think it's okay to name it more broadly, and then refactor the sklearn wrapper to use it later on.

Reply:
The issue right now is this is relied on to distinguish between needing a SklearnTrainingConfig or KerasTrainingConfig in my other PR. This can be generalized if/when I add support for model-level training configuration class definitions.

This currently exists in shared because it must exist in `config.py`. I could move that file to the top level outside of shared, instead?

Reply:
The top-level would also be shared too IMO.

I think the easy move is to rename it to `register_estimator`, and then use it for the sklearn models too in a later PR once you merge the keras and sklearn factories.

It's okay to if sklearn doesn't use some general purpose shared code, but it is confusing to put code that can only be used for to constructing keras models in a shared location.